src/stackTools/fusionRenderer.js

```diff
@@ -31,9 +31,9 @@ export default class FusionRenderer {
         const currentLayerId = this.layerIds[0];
         const layer = cornerstone.getLayer(element, currentLayerId);
 
-        layer.image = image;
+        layer.image = $.extend({}, image);
```

Comment:
Could we do this without jQuery instead? Given that we still eventually plan to remove it.

Reply:
The ES6 equivalent is Object.assign(), but not IE compatible (and honestly I don't think it's a good idea to remove jQuery).

Reply:
I didn't see there is already many Object.assign() in the code.

Reply:
done :)